### PR TITLE
check docversion in module/blockrange requests

### PR DESCRIPTION
**Pull request by pfitzseb**
Requires https://github.com/julia-vscode/LanguageServer.jl/pull/778.

Should fix a few of the errors in the crash reports.

### CODE REVIEW

File: src/interactive/modules.ts [hunk 1]
```diff
@@ -90,12 +91,14 @@ async function updateModuleForSelectionEvent(event: vscode.TextEditorSelectionCh
...
 
-    let loaded = false
-    if (isConnectionActive()) {
-        loaded = await g_connection.sendRequest(requestTypeIsModuleLoaded, mod)
-    }
+    if (mod) {
+        let loaded = false
+        if (isConnectionActive()) {
+            loaded = await g_connection.sendRequest(requestTypeIsModuleLoaded, mod)
+        }
```
Comment (davidanthoff) [inline src/interactive/modules.ts:98]: I thought we were handling these situations with `try catch` now? I think we should, the `isConnectionActive()` pattern doesn't work in async code.

Comment (pfitzseb) [reply src/interactive/modules.ts]: True.

**Comment by pfitzseb** [comment]
The various `console.error(err)`s should probably be patched into crash reporting somehow (depending on whether it's an expected error or not, I suppose).

**Comment by davidanthoff** [comment]
> probably be patched into crash reporting somehow

You could use `traceEvent` or `traceTrace` for now, I think we just want to know whether we still see these conditions, right?

But I'd say we can merge this now and then add this in a new PR?

**Comment by davidanthoff** [comment]
I'm going to merge the underlying LS PR and this one, given that we need a submodule update in-between it is probably easier if I do the whole merge story.

**Pull request merged**